libs/minecraft/PosUtils.arucas

```diff
@@ -45,26 +60,54 @@ enum Direction {
         return Direction.NONE;
     }
 
-    fun vec() {
+    fun rotateYClockwise() {
+        return this.right();
+    }
+
+    fun rotateYCounterclockwise() {
+        return this.left();
+    }
+
+    fun getAxis(){
+        return this.axis;
+    }
+
+    fun asVector() {
         return this.vector.copy();
     }
 
-    fun x() {
+    fun unitVec(){
+        return this.vector.copy();
+    }
+
+    fun getX() {
         return this.vector.getX();
     }
 
-    fun y() {
+    fun getY() {
         return this.vector.getY();
     }
 
-    fun z() {
+    fun getZ() {
         return this.vector.getZ();
     }
 
     fun toString() {
         return this.getName().lowercase();
     }
 
+    fun isValid(){
+        return this != Direction.NONE;
+    }
+
+    fun mult(value){
+        return this.vector.multiply(value);
+    }
+
+    fun multiply(value){
```

Comment:
Do we need `mult` and `multiply`? That's slightly confusing I would rather just keep `multiple` then it's consistent with `Pos` class.

Reply:
its actually from minecraft reference, just for convenience

Reply:
I think it's more confusing and one should be removed